src/Migrations/DeleteColumnMigration.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Abagayev\Laravel\MigrationShortcuts\Migrations;
+
+abstract class DeleteColumnMigration extends AddColumnMigration
+{
+    /** @var bool If you need to recreate column in down method */
+    protected $recreateColumn = false;
```

Comment:
Don't think that this param is needed, it's a good practice to always create column at rollback